packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationUtils.kt

```diff
@@ -0,0 +1,40 @@
+package expo.modules.localization
+
+import android.text.TextUtils
+import java.lang.Exception
+import java.util.*
+import kotlin.collections.ArrayList
+
+val USES_IMPERIAL = listOf("US", "LR", "MM")
+
+val iSOCurrencyCodes: ArrayList<String> by lazy {
+  Currency.getAvailableCurrencies().map { it.currencyCode } as ArrayList<String>
+}
+
+fun getLocaleNames(locales: ArrayList<Locale>?) = locales?.map { it.toLanguageTag() } as ArrayList<String>
+
+fun getCountryCode(locale: Locale): String? {
+  return try {
+    val country = locale.country
+    if (TextUtils.isEmpty(country)) null else country
+  } catch (ignored: Exception) {
+    null
+  }
+}
```

Comment:
```suggestion
fun getCountryCode(locale: Locale): String? {
  return runCatching {
    val country = locale.country
    if (TextUtils.isEmpty(country)) null else country
  }.getOrNull()
}
```

We can use `runCatching` similarly as in `getSystemProperty` method, I believe